### PR TITLE
Quiz template

### PR DESCRIPTION
**Pull request by sri-rishi**
Added Quiz Question Template

### CODE REVIEW

File: quiz-template.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+</head>
+<body>
+    <nav class="nav-box nav-div flex-row align-center">
+        <div class="nav-section flex-row align-center justify-evenly">
+            <div class="hamburger display-none btn-border-none bg-transparent">
+                <i class="fas fa-bars"></i>
+            </div>
+            <div class="brand-title">
+                <a href="#">
+                    <div class="title font-weight-3">
```
Comment (Ankur9669) [inline quiz-template.html:20]: fix: added link to index.html here

File: quiz-template.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+                <p>Question: 1/5</p>
+                <p>Score: -10</p>
+            </div>
+            <p class="xsm-heading">What is the favorite camera choice among professional bird photographers?</p>
+            <div class="flex-column gap-1">
+                <button class="ans bg-transparent right-ans card-shadow">Canon</button>
+                <button class="ans bg-transparent">Nikon</button>
+                <button class="ans bg-transparent wrong-ans card-shadow">Sony</button>
+                <button class="ans bg-transparent">None the above</button>
+            </div>
```
Comment (Ankur9669) [inline quiz-template.html:72]: fix: I think these will be radio buttons not simple buttons

**Review by Ankur9669 — CHANGES_REQUESTED** [review]
The code looks good, I have suggested some changes

**Review by JuvitaSaini09 — APPROVED** [review]
code looks good to merge

**Review by manish-GH — APPROVED** [review]
The UI looks clean. As Ankur suggested change the buttons to radio buttons, then you are good to merge

File: quiz-template.html (hunk 1 above)
Comment (sri-rishi) [reply quiz-template.html:20]: will add in end because of file structure

File: quiz-template.html (hunk 2 above)
Comment (sri-rishi) [reply quiz-template.html:72]: done

**Review by Ankur9669 — APPROVED** [review]

**Pull request merged**